### ISSUE TITLE
Should TAG appointees be automatically nominated for the next TAG election when their term expires?

### ISSUE THREAD

**Issue by dwsinger** [w3c/w3process#166]
It seems that if they are willing to serve, we should see if they can be elected, and then the Director can continue to find balance and new people to appoint. If one of the difficulties in getting elected is being unknown, or people unsure how you'll do, surely a term of appointment mitigates that (and if you didn't get better known as a result of your appointment, that is also a point of interest).

I think the Director should still be able to appoint people who fail to get elected, but that community feedback on those candidates could be input to Directorial consideration.

**Comment by tantek** [comment]
While this is also vulnerable to the "don't want to run against the other (politically stronger) candidates and be embarrassed for not being elected and having to be appointed, again" problem, I agree with the general sentiment and we should figure out a way to make this work. I think term limits #165 would help this issue as well (soft dependency).

**Comment by vfournier17** [comment]
I agree that term limits would not be a good thing.  We might end up with completely novice members, or no members if others will not agree to take on this significant responsibility.   Losing an election is part of the risk of running - people have to take that into consideration when they throw their hats into the ring.  Also, I've seen people in other SDOs send out emails about their experience, positions, etc., which could help if they're relatively unknown.

**Comment by dwsinger** [comment]
We'd like this discussed in the AB, TAG and with the Director, before we do the (easy) job of editing the process...

**Comment by jeffjaffe** [comment]
Any appointee that is not re-appointed should have no trouble getting nominated from someone.  Why make this automatic?

**Comment by dwsinger** [comment]
Why bother to nominate when the Director can simply re-appoint? I think that having them automatically stand 
* enables the Director initially to make people 'more visible' when they're good but not well known
* enables the community to give feedback (the Director might think harder about re-nominating someone who didn't get voted in)
* enables the Director to have a free-er hand when appointing to re-balance

**Comment by chaals** [comment]
It would be helpful to have input from @torgo and @plinss on this, as they would have been directly affected several times already. My sense is that with the current practice of Director's appointments being used to provide the chairs, this would be a destabilising change with little apparent upside.
